packages/cli/test/commands/profile.test.ts

```diff
@@ -30,14 +29,9 @@ describe("profile: no url, no cookies", () => {
   test
     .stdout()
     .it(
-      `runs profile ${HAR_PATH} --tbResultsFolder ${TB_RESULTS_FOLDER} --hideUsertimings`,
+      `runs profile ${HAR_PATH} --tbResultsFolder ${TB_RESULTS_FOLDER}`,
```

Comment:
does it still use the HAR for anything?

Reply:
Yes for `traceJSONPath` which is the HAR

https://github.com/TracerBench/tracerbench/blob/master/packages/cli/src/commands/profile.ts#L283-L342